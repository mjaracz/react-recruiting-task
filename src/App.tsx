import React from 'react';
import {Provider} from 'react-redux'
import {store} from './redux/store'
import {Routing} from './routing'

function App() {
  return (
    <Provider store={store}>
      <Routing/>
    </Provider>
  );
}

export default App;
