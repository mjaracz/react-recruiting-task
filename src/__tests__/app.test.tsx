import React from 'react'
import {render} from '@testing-library/react'
import App from '../App'
import {BrowserRouter} from 'react-router-dom'
import {StoreProvider} from '../storeProvider'

describe('<App/>', () => {
  it('should be defined and match snapshot', function () {
    const {baseElement} = render(
      <BrowserRouter>
        <StoreProvider>
          <App/>
        </StoreProvider>
      </BrowserRouter>
    )
    expect(baseElement).toMatchSnapshot()
  });
});
