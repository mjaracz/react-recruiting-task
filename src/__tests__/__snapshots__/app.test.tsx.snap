// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should be defined and match snapshot 1`] = `
<body>
  <div>
    <div>
      <nav
        class="makeStyles-root-1"
      >
        <a
          class="makeStyles-barItem-2"
          href="/absences"
        >
          absences
        </a>
        <a
          class="makeStyles-barItem-2"
          href="/members"
        >
          members
        </a>
      </nav>
      <div
        class="makeStyles-root-3"
      >
        <div
          class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
          role="progressbar"
          style="width: 80px; height: 80px;"
        >
          <svg
            class="MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</body>
`;
